main.tf

```diff
@@ -1,30 +1,29 @@
-provider aws {}
-
 provider aws {
   alias = "ns"
 }
 
+locals {
+  create_route53_delegation = false
```

Comment:
how will a user override these values, when they _do_ want to create the delegation or query log? everything else looks fine, but the user needs to be able to enable these modules, and they can't if we change to use locals.